recipes/qt/5.x.x/conanfile.py

```diff
@@ -646,6 +656,9 @@ def _getenvpath(var):
     def _cmake_executables_file(self):
         return os.path.join("lib", "cmake", "Qt5Core", "conan_qt_executables_variables.cmake")
 
+    def _cmake_qt5_private_file(self, module):
+        return os.path.join("lib", "cmake", "Qt5{0}".format(module), "conan_qt_qt5_{0}private.cmake".format(module.lower()))
```

Comment:
This is never called. is it needed ?

Reply:
You are right, I will remove that function.